### ISSUE TITLE
A JavaScript error occurred when changing options in Preferences

### ISSUE THREAD

**Issue by motchie** [ThePacielloGroup/CCAe#175]
**Describe the bug**
A JavaScript error occurred when changing options in Preferences.

**Desktop (please complete the following information):**
 - OS: Mac OS X 10.15.5 19F101
 - Version: 3.1.0

**To Reproduce**
1. Open `Preferences` menu from `Colour Contrast Analyser (CCA)`
2. Change some values in Options group. Not in Shortcuts.
3. Press `Save` button.
4. JavaScripe error message dialog apperd saying:
```
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: i18n.lang is not a function
    at Object.<anonymous> (/Applications/Colour Contrast Analyser (CCA).app/Contents/Resources/app.asar/src/CCAcontroller.js:222:76)
    at Array.forEach (<anonymous>)
    at CCAController.updateContrastRatio (/Applications/Colour Contrast Analyser (CCA).app/Contents/Resources/app.asar/src/CCAcontroller.js:218:53)
    at CCAController.setPreference (/Applications/Colour Contrast Analyser (CCA).app/Contents/Resources/app.asar/src/CCAcontroller.js:318:26)
    at IpcMainImpl.emit (events.js:223:5)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:3911:15)
    at WebContents.emit (events.js:223:5)
```

**Expected behavior**
Changed options are saving normally.

**Screenshots**
None.

**Additional context**
None.

**Comment by ferllings** [comment]
Thanks, just noticed it. I'll fix that quickly.

**Comment by motchie** [comment]
@ferllings I confirmed this issue fixed in 3.1.1. Should I close this issue?
